src/dotnet/commands/dotnet-publish/LocalizableStrings.cs

```diff
@@ -29,5 +29,9 @@ internal class LocalizableStrings
         public const string VersionSuffixOption = "VERSION_SUFFIX";
 
         public const string VersionSuffixOptionDescription = "Defines the value for the $(VersionSuffix) property in the project.";
+
+        public const string FilterProjOption = "profile.proj";
+
+        public const string FilterProjOptionDescription = "The list of packages to be excluded from publish step.";
```

Comment:
This description does not seem right. You say it is a list of packages, but you pass to it a .proj file. Should it say a project file containing the list of packages to be excluded...?